packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeFeeRegistry.sol

```diff
@@ -44,7 +44,11 @@ contract AirnodeFeeRegistry is
         override prioritizeEndpointPriceOverChainPrice;
 
     /// @notice Prices will have upto 18 decimal places
-    uint8 public constant DECIMALS = 18;
+    uint8 public constant override DECIMALS = 18;
+
+    /// @notice Prices are represented in seconds for 30 day intervals
+    /// @dev interval will be counted in days
+    uint24 public constant override INTERVAL = 30 days;
```

Comment:
I'm thinking that maybe we may need to make this value configurable per Airnode and default to this constant if none is specified.

Reply:
Yeah I think that would probably be better considering we are having everything else configurable.